cli/cmd/errors.go

```diff
@@ -130,6 +131,20 @@ func ErrorInvalidOperatorEndpoint(endpoint string) error {
 	})
 }
 
+// err can be passed in as nil
+func ErrorNoOperatorLoadBalancer(envName string, err error) error {
+	errMsg := fmt.Sprintf("unable to locate operator load balancer; you can attempt to resolve this issue and configure your CLI environment by running `cortex cluster info --env %s`", envName)
+
+	if err != nil {
```

Comment:
This doesn't look like its being used anywhere so I think this code can be removed for now. Also I think that if I saw an invocation of ErrorNoOperatorLoadBalancer() I would expect the return error kind to be `ErrNoOperatorLoadBalancer` but in this case, it's possible that the result is a different error kind with the message modified.. We can discuss this error pattern if you think it is important

Reply:
Agreed